website/content/v1.3/kubernetes-guides/configuration/storage.md

```diff
@@ -107,6 +107,79 @@ talosctl -n <node ip> service kubelet restart
 
 Continue setting up [Mayastor](https://mayastor.gitbook.io/introduction/quickstart/deploy-mayastor) using the official documentation.
 
+### Piraeus / LINSTOR
+
+* [Piraeus-Operator](https://piraeus.io/)
+* [LINSTOR](https://linbit.com/drbd/)
+* [DRBD Extension](https://github.com/siderolabs/extensions#storage)
+
+<!-- markdownlint-disable MD007 -->
+<!-- markdownlint-disable MD032 -->
+<!-- markdownlint-disable MD025 -->
+
+{{< tabpane lang="yaml" >}}
+{{< tab header="Strategic merge patch" >}}
+machine:
+  install:
+    extensions:
+      - image: ghcr.io/siderolabs/drbd:v1.3.0
+  kernel:
+    modules:
+      - name: drbd
+      - name: drbd_transport_tcp
+{{< /tab >}}
+{{< tab header="JSON patch" >}}
+- op: add
+  path: /machine/install/extensions/-
+  value:
+    - image: ghcr.io/siderolabs/drbd:v1.3.0
```

Comment:
should be ghcr.io/siderolabs/drbd-pkg:v1.3.0

Reply:
good catch!
-> fixed